aquarius/events/events_monitor.py

```diff
@@ -282,10 +285,15 @@ def get_last_processed_block(self):
             block = (
                 last_block_record["last_block"]
                 if last_block_record["last_block"] >= 0
-                else 0
+                else get_defined_block(self._chain_id)
             )
         except Exception as e:
-            logging.error(f"Cannot get last_block error={e}")
+            # Retrieve the defined block.
+            if type(e) == elasticsearch.NotFoundError:
```

Comment:
not sure this is correct.

Reply:
Why? if the error is `NotFoundError` from ES, should retrieve the `defined_ block`